start.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/bash
+
+# Add nagios dirs
+mkdir -p /var/log/nagios/archives \
+      /var/log/nagios/rw \
+      /var/log/nagios/spool/checkresults
+
+# Add nagios user
+htpasswd -c -b -s /etc/nagios/passwd ${NAGIOS_USER} ${NAGIOS_PASSWORD}
+
+# Generate command config
+python /opt/rhmap/make-nagios-commands-cfg.py
+
+# Generate fhservices config
+python /opt/rhmap/make-nagios-fhservices-cfg.py
+
+/usr/bin/supervisord
```

Comment:
```
exec /usr/bin/supervisord
```

I'm not sure `supervisord` works fine with Docker. For sure we should exec it to make it run as PID 1, otherwise the shell won't handle signals properly / won't forward signals to `supervisord`.